### ISSUE TITLE
[FEATURE] Implement Image Recognition Using TensorFlow in .NET

### ISSUE THREAD

**Issue by guibranco** [GuilhermeStracini/POC-dotnet-TensorFlow#4]
#### Overview
This repository will serve as a Proof-of-Concept (PoC) for a .NET project that demonstrates image recognition using TensorFlow. The goal is to build a simple application capable of recognizing objects in images by following the steps outlined in [this tutorial](https://medium.com/@paulotorres/real-time-image-recognition-with-net-core-and-tensorflow-5b03743e0116).

#### Tasks
1. **Set up TensorFlow in .NET**
   - Install required NuGet packages: `TensorFlow.NET` and `SciSharp.TensorFlow.Redist`.
   
2. **Preprocess Images**
   - Implement image loading and preprocessing logic.
   
3. **Build Model Integration**
   - Load the pre-trained TensorFlow model and integrate it into the .NET project.
   
4. **Perform Image Recognition**
   - Execute image recognition on a sample image and output results.

#### Example Code

Below is a sample snippet in C# that shows how to load a TensorFlow model and use it for image recognition:

```csharp
using System;
using Tensorflow;
using Tensorflow.NumPy;

class Program
{
    static void Main(string[] args)
    {
        // Load TensorFlow model
        var graph = new Graph().as_default();
        var modelPath = "path_to_your_model.pb";
        var session = new Session(graph);
        graph.Import(modelPath);

        // Prepare image input
        var image = LoadImage("path_to_your_image.jpg");

        // Run the model
        var output = session.run(new[] { "output_node" }, new FeedItem[] { new FeedItem("input_node", image) });

        // Process and display results
        Console.WriteLine($"Predicted: {ProcessOutput(output)}");
    }

    static NDArray LoadImage(string imagePath)
    {
        // Implement image loading and preprocessing logic here
        return new NDArray();
    }

    static string ProcessOutput(NDArray output)
    {
        // Implement output processing logic here
        return "Object name";
    }
}
```

This code shows how to:
1. Load a pre-trained TensorFlow model.
2. Feed an image into the model for recognition.
3. Process the output to display recognized objects.

#### Resources
- [TensorFlow.NET Documentation](https://github.com/SciSharp/TensorFlow.NET)
- [Original Tutorial](https://medium.com/@paulotorres/real-time-image-recognition-with-net-core-and-tensorflow-5b03743e0116)

**Comment by guibranco** [comment]
@gstraccini create labels
